modules/hapibd/meta.yml

```diff
@@ -0,0 +1,67 @@
+name: "hapibd"
+description: The hap-ibd program detects identity-by-descent (IBD) segments and homozygosity-by-descent (HBD) segments in phased genotype data. The hap-ibd program can analyze data sets with hundreds of thousands of samples.
+keywords:
+  - ibd
+  - hbd
+  - beagle
+
+tools:
+  - "hapibd":
+      description: "Hap-ibd Detects identity-by-descent (IBD) segments and homozygosity-by-descent (HBD) segments in phased genotype data."
+      homepage: "https://github.com/browning-lab/hap-ibd/blob/master/README.md"
+      documentation: "https://github.com/browning-lab/hap-ibd/blob/master/README.md"
+      tool_dev_url: "None"
+      doi: "https://doi.org/10.1016/j.ajhg.2020.02.010"
+      licence: "['Apache-2.0']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing cohort information
+        e.g. [ id:'test' ]
+
+  - vcf:
+      type: file
+      description: phased VCF file with a GT FORMAT subfield with no missing alleles
+      pattern: "*.{vcf,vcf.gz}"
+  - map:
+      type: file
+      description: genetic map with cM units in PLINK format
+      pattern: "*.{map,map.gz,map.zip}"
+  - out:
+      type: string
+      description: the output filename prefix
+      pattern: "example: cohort.hapibd.out"
```

Comment:
It seems like this is not one of the inputs anymore

Reply:
You're right, I removed this!